packages/framework/client-logger/external-telemetry/api-extractor-cjs.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "../../../../common/build/build-common/api-extractor-base.cjs.primary.json",
+	// CJS is actually secondary; so, no report.
+	"apiReport": {
+		"enabled": false
+	},
+	"messages": {
```

Comment:
Since this is a new package, and small. I would recommend adding the following block under `messages` (to both of the non-lint configs):
```json
"extractorMessageReporting": {
	// Require documentation on all package-exported API items.
	"ae-undocumented": {
		"logLevel": "error",
		"addToApiReportFile": false
	}
```

This will ensure that API extractor will fail if we add any new API members without documentation 🙂

Reply:
Added in commit 20a366b